module-bsp/CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ target_sources(
         bsp/common.cpp
         bsp/lpm/bsp_lpm.cpp
         devices/Device.cpp
+        devices/power/CW2015.cpp
+        devices/power/MP2639B.cpp
```

Comment:
Please move it to bell-specific directory

Reply:
These are no longer product-specific so I can't see any reason to put them to bell.

Reply:
In this cmakelists there are mainly cpp wrappers for vendor peripheral drivers. For me it does not fit so well here since I cannot see here other product-specific drivers. It may be decoupled from the product logic but it still could be found only in Bell HW.

Reply:
These drivers only appear in Bell because currently are only used by the Bell. It is still better to have a pool of specific device drivers designed that way they are not pinned to the specific board. As a matter of fact, for instance, `MAX77818` doesn't care on which platform it is used. It only requires an I2C instance and if a user wants to use IRQ, he must provide an interrupts environment prepared ahead. That's where `hal` comes into the game. Every board must implement board-specific HAL. Ideally, even boards wouldn't be pinned to the specific application(in our codebase it would be hard to achieve it without some refactor).